example-config.toml

```diff
@@ -2,6 +2,9 @@ FsApiToken = "<your FullStory API token>"
 Backoff = "30s"
 BackoffStepsMax = 8
 
+# SegmentId is the string identifier for the segment you want to use with Hauser. 
+SegmentId = "<your segment id>"
```

Comment:
This value is not required and this does not result in a valid config, so I would comment this line out.

As far as a comment, is there a help doc somewhere referencing where a segment ID can be retrieved in the UI? Maybe point to that, or add a section in the README about how to determine this? I don't think it's necessary, but could help with discovery.

Reply:
I revised the toml changes and added a section to the README. Let me know if it needs additional tweaking.